src/components/memoBlock/memo.component.tsx

```diff
@@ -76,13 +92,38 @@ function MemoBlock(props: Props): JSX.Element {
       }, 1000);
     }
   }
+
+  const levels = ["3*4", "4*4", "4*5", "4*6", "4*7", "4*8"];
+
+  const parseLevel = (level: string) => {
+    const [row, col] = level.split("*");
+    return {
+      row: parseInt(row),
+      col: parseInt(col),
+    };
+  };
```

Comment:
This function is used only on the handler of levels. Move it inside the handler so you avoid exposing implementation details.

```suggestion
```